peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+PEP: 772
+Title: A Python Installation Manager for Windows
+Author: Steve Dower
+PEP-Delegate: TBD
+Discussions-To: https://discuss.python.org/t/pep-772-a-python-installation-manager-for-windows/77900
+Status: Draft
+Type: Standards Track
+Topic: Release
+Created: 21-Jan-2025
+Post-History: 18-Dec-2024
+
+
+Abstract
+========
+
+Installation of the python.org Python distribution on Windows is complex.
+There are three main approaches with roughly equivalent levels of user
+experience, and yet all of these suffer from different limitations, including
+failing to satisfy modern usage scenarios. This PEP proposes a design for
+a single Windows install workflow tool that satisfies all the needs of the
+existing installers for the platform, while avoiding most of their limitations,
+and provides the core team with the ability to manage releases for many years
+to come.
+
+Reader's Note
+-------------
+
+This document is a detailed *proposal*, intended to assist in an "approve" or
+"reject" decision on the replacement of the existing install formats for Python
+on Windows with a new format. It is not intended to be a binding specification,
+nor is it user documentation or an interoperability specification. The actual
+implementation may vary over time as needs change, and that is not to be
+considered a "violation" of this PEP. Any interfaces and protocols meant for
+public use will be independently documented and maintained in accordance with
+standard deprecation timelines.
+
+Documentation on installing Python on Windows can be found at
+`docs.python.org/using/windows.html <https://docs.python.org/using/windows.html>`__.
```

Comment:
We can use Sphinx roles here. We could also change the link text to something more meaningful than just the URL?
```suggestion
:ref:`docs.python.org/using/windows.html <using-on-windows>`.
```

Reply:
The point is to show the exact URL, so that people feel confident linking it from other places. Otherwise I've seen some people get nervous that they won't be able to point people towards documentation.